modules/storages/lib/open_project/storages/append_storages_hosts_to_csp_hook.rb

```diff
@@ -48,11 +48,19 @@ def application_controller_before_action(context)
                                                          .select(:storage_id)
     hosts = ::Storages::Storage.where(id: projects_with_permission)
                                .pluck(:host)
+                               .map(&method(:remove_uri_path))
```

Comment:
@Kharonus Just a curious question: What happens if the path is path stays present? CSP can't handle that?